src/components/Card.js

```diff
@@ -28,18 +42,37 @@ class Card {
     });
   }
 
-  _handleLikeButton = () => {
-    this._likeButton.classList.toggle("card__like-button_active");
-  };
-
-  _handleDeleteCard() {
+  handleDeleteCard() {
     this._cardElement.remove();
     this._cardElement = null;
   }
 
+  updateLikes(likes) {
+    const likeButton = this._cardElement.querySelector(".card__like-button");
```

Comment:
You've already found `this._likeButton` in line 76. Please, use it everywhere

`Class fields` (the so-called **class variables** with `this`) are accessible inside any class method. You shouldn’t pass **class fields** into the call of any method or search them again because you can use class fields inside all methods.